### DISCUSSION TITLE
Need help creating sprites

### DISCUSSION THREAD

**Discussion by tails1154** [haven1433/HexManiacAdvance#203] [Q&A] [answered]
I need to create sprite data in HMA and Don't know how to do It. I have looked everywhere for it and nothing on youtube says anything about it or on the wiki. Any help would be greatly appreciated. Have a nice day! :)

**Comment by haven1433** [comment, marked as answer]
What kind of sprite? Generally you want to start from a table that contains the sprites that you want. For example, if you want to make a new item sprite, start at the table of item sprites.

From there, if no data exists you can click a "create new" button to make new "blank" data automatically. Or if you have two pointers to the same image, you can repoint one to make a copy of it. Once you have the data, you can import an image, or edit it directly in the program.

**Reply by tails1154** [reply]
thank you so much! I just repointed it and it worked! Thank you! :)
